software/keycloak.php

```diff
@@ -0,0 +1,98 @@
+<?php
+require_once __DIR__ . '/abstract.php';
+include_once(__DIR__ . '/../methods/github.php');
+
+//https://docs.github.com/en/rest/reference/repos#list-releases
+
+class rabbitmq extends SoftwareCheck
+{
+    public static $name = 'Keycloak';
+    public static $vendor = 'RedHat';
+    public static $homepage = 'https://www.keycloak.org';
+    public static $type = 'github';
+    public static $enabled = true;
+    var $owner = 'keycloak';
+    var $repo = 'keycloak';
+    var $regex = '/^v(\d+\.\d+\.\d+).*/';
```

Comment:
Their releases aren't prefixed with a "v" (eg. v1.2.3), it's just 1.2.3.
```suggestion
    var $regex = '/^(\d+\.\d+\.\d+).*/';
```